raitools/raitools/raianalyzer/raianalyzer.py

```diff
@@ -135,12 +156,85 @@ def list(self):
+            with open(Path(path) / SERIALIZER, 'wb') as file:
+                pickle.dump(self._serializer, file)
+            # save the model
+            with open(Path(path) / MODEL_PKL, 'w') as file:
+                file.write(model_data)
+        else:
+            has_setstate = hasattr(self.model, '__setstate__')
+            has_getstate = hasattr(self.model, '__getstate__')
+            if not (has_setstate and has_getstate):
+                raise ValueError(
+                    "Model must be picklable or a custom serializer must"
+                    " be specified")
+            with open(Path(path) / MODEL_PKL, 'wb') as file:
+                pickle.dump(self.model, file)
 
     @staticmethod
     def load(path):
-        # TODO: implement
-        raise NotImplementedError(
-            "Load not implemented for RAIAnalyzer")
+        """Load the RAIAnalyzer from the given path.
+
+        :param path: The path to load the RAIAnalyzer from.
+        :type path: str
+        """
+        # create the RAIAnalyzer without any properties using the __new__
+        # function, similar to pickle
+        inst = RAIAnalyzer.__new__(RAIAnalyzer)
+        top_dir = Path(path)
+        # load current state
+        train = pd.read_pickle(top_dir / TRAIN)
+        inst.__dict__[TRAIN] = train
+        inst.__dict__[TEST] = pd.read_pickle(top_dir / TEST)
+        with open(top_dir / META_JSON, 'r') as meta_file:
+            meta = meta_file.read()
+        meta = json.loads(meta)
+        target_column = meta[TARGET_COLUMN]
+        inst.__dict__[TARGET_COLUMN] = target_column
+        inst.__dict__[TASK_TYPE] = meta[TASK_TYPE]
+        inst.__dict__['_' + CLASSES] = train[target_column].unique()
+        serializer_path = top_dir / SERIALIZER
+        if serializer_path.exists():
+            with open(serializer_path) as file:
+                serializer = pickle.load(file)
+            inst.__dict__['_' + SERIALIZER] = serializer
+            with open(top_dir / MODEL_PKL, 'rb') as file:
+                inst.__dict__[MODEL] = serializer.load(file)
+        else:
+            inst.__dict__['_' + SERIALIZER] = None
+            with open(top_dir / MODEL_PKL, 'rb') as file:
+                inst.__dict__[MODEL] = pickle.load(file)
+        # load each of the individual managers
+        manager_names = ManagerNames.get_managers()
+        managers = []
+        kvp_manager = {ManagerNames.CAUSAL: CausalManager,
+                       ManagerNames.COUNTERFACTUAL: CounterfactualManager,
+                       ManagerNames.ERROR_ANALYSIS: ErrorAnalysisManager,
+                       ManagerNames.EXPLAINER: ExplainerManager,
+                       ManagerNames.FAIRNESS: FairnessManager}
+        for manager_name in manager_names:
+            manager_cls = kvp_manager[manager_name]
+            manager = manager_cls.load(top_dir / manager_name, inst)
+            _manager_name = manager_name.replace(' ', '_')
```

Comment:
This feels like its adding an assumption which could trip us up later

Reply:
It's just for "error analysis" manager name.  If you prefer, I can change it to "error_analysis" instead and then this can be removed.

Reply:
I changed it to error_analysis